src/main/java/in/ac/bits/protocolanalyzer/persistence/repository/AnalysisRepository.java

```diff
@@ -126,4 +127,12 @@ private void publishHigh() {
 		log.info(System.currentTimeMillis());
 		eventBus.post(new BucketLimitEvent("STOP"));
 	}
+	
+	public boolean getIsFinished() {
```

Comment:
Lombok provides getter and setter methods. We don't need these two methods.

Reply:
as the field is named as "isFinished" and is of type boolean lombok doesn't create its setter/getter.
Ex: private boolean answer;
the getter method created by lombok is isAnswer().
So if a boolean data member is of the form "is...." lombok skips creating getter and setter methods.
Solution: Rename the field or make it public or create your own setter and getter.

Reply:
We can rename the variable to `finished`